src/tests/Loader/classloader/InlineArray/InlineArrayValid.cs

```diff
@@ -109,7 +106,7 @@ struct Arr1
         public E e;
 
         [UnscopedRef]
-        public ref E this[int i] => ref Unsafe.Add(ref e, i);
+        public ref E At(int i) => ref Unsafe.Add(ref e, i);
```

Comment:
nit: `Arr1` doesn't need this `At` function at all. All the tests (as far as i could tell) use indexing - which roslyn generates.